packages/interactivity-router/src/head.ts

```diff
@@ -29,6 +37,14 @@ export const updateHead = async ( newHead: HTMLHeadElement[] ) => {
 		}
 	}
 
+	await Promise.all(
+		[ ...headElements.entries() ]
```

Comment:
**This looks good**, but I always worry about these data structures because I think of Map as an unordered collection. I was worried about dependencies and ordering not being respected:

> The `entries()` method of Map instances returns a new map iterator object that contains the [key, value] pairs for each element in this map *in insertion order*.

[From MDN (bold mine)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/entries).

It seems like we're OK since entries returns elements in insertion order 👍